src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
The selector `.stars__star` is styling by tag name, which is not recommended according to the checklist. Consider using a class instead to apply these styles.